test.js

```diff
@@ -50,6 +47,77 @@ const testnumber = () => {
   const result2 = 4;
   console.log(' Print the number, if it is not divisible by 3 or 5 or both'+ '\n'+
     assertEqual(result, result2, 4));
-
 }
 testnumber();
+
+const errorNull = () => {
+  const result = tools.fizzbuzz();
+  const result2 = {status: 403, message: 'Sorry, you passed null value'};
```

Comment:
Now that you have errorCodes.js, you can reference that here, instead of hand-writing it